### DISCUSSION TITLE
Could not connect to host ... after upgrade

### DISCUSSION THREAD

**Discussion by Coooz** [nextgenhealthcare/connect#5185] [Q&A]
Hi,

For a customer of ours I have upgraded Mirth Connect from 3.5.1 to 3.12. One channel has a TCP Sender destination that worked fine in 3.5.1, but which fails to connect in 3.12: the Test Connection button gives a message "Could not connect to host: ..."
I upgraded by closely following the official upgrade guide: https://github.com/nextgenhealthcare/connect/wiki/Upgrading. What I did not do, however, was upgrade Java. This is version 1.8.0_191. Perhaps this is relevant.

What can I do to obtain a working connection? Thanks!

**Comment by pacmano1** [comment]
There shouldn't be anything preventing that.   Have you attempted standard troubleshooting steps by attempting the connection outside of Mirth?   Using tools such as telnet?  netcat?  

Are you sure Mirth had stopped and restarted during the upgrade process?

**Reply by Coooz** [reply]
Hi pacmano1,

It is working now... it seemed to have to do with an errored message blocking the message queue. Thanks for replying!
